### PR TITLE
#324 fixed - derivation of phpunit runner pointing to php instead of binary

### PR DESCRIPTION
**Pull request by mmeuser**
before:
used the composer shell script to run phpunit

now:
uses the phpunit php start script to run phpunit

### CODE REVIEW

**Comment by michaelgv** [comment]
Looks good! Can you resolve the conflicts?

**Comment by mmeuser** [comment]
Conflicts are resolved.

File: test/bootstrap.php [hunk 1]
```diff
@@ -18,10 +18,7 @@
 define('PARATEST_ROOT', dirname(__DIR__));
 
 //check for .bat first if on windows.
```
Comment (andreasschroth) [inline test/bootstrap.php]: From what I can see, you don't check for the phpunit.bat file anymore on Windows, so this comment is outdated and should be removed.

**Comment by andreasschroth** [comment]
Could you please add an explanation of the purpose of this change? What advantages does this one have over using the bin via `vendor/bin`? Now the folder of phpunit is hard-coded, let's say (probably not going to happen, but just an example), Sebastian changes the phpunit namespace from `phpunit` to `sebastianbergmann`, then this is going to fail.

**Comment by andreasschroth** [comment]
Is this related to #332 ?

**Comment by andreasschroth** [comment]
Okay, I can now see it's related to your issue #324 ... Well, if using phpunit directly fixes the Windows problem, then this is a great PR. 👍 

Thank you for your contribution, I'm going to merge this now. Next time, it would only be great if you could provide some more context to the PR or link to a specific issue.

**Comment by andreasschroth** [comment]
Okay, I can now see it's related to your issue #324 ... I didn't see it at first, because it was only included in the PR title and not in the description. Well, if using phpunit directly fixes the Windows problem, then this is a great PR. 👍 

Thank you for your contribution, I'm going to merge this now.

**Pull request merged**